ouster_ros/src/os_cloud_node.cpp

```diff
@@ -67,8 +70,18 @@ int main(int argc, char** argv) {
                 });
             if (h != ls.headers.end()) {
                 scan_to_cloud(xyz_lut, h->timestamp, ls, cloud);
+
+                // set ROS time if sensor is not yet synchronized via PTP
+                std::chrono::nanoseconds ts_data = h->timestamp;
+                std::chrono::seconds h_seconds = std::chrono::duration_cast<std::chrono::seconds>(h->timestamp);
+                if ( h_seconds.count() < 100000 && sensor::TIME_FROM_PTP_1588 == timestamp_mode_int ) {
```

Comment:
perhaps check distance from ros::Time::now() in seconds in stead of a magic 100.000